Makefile

```
@@ -237,38 +261,43 @@ kubescape-orig:
 
 
+# NOTE: node-agent is NEVER restarted by any target here. It must come up once,
+# on its own, with the right config already in place — hence the post-renderer
+# below instead of a patch-then-bounce. Do not reintroduce `rollout restart ds
+# node-agent`. Also do NOT pass --set nodeAgent.privileged=true: the chart
+# default is false and privileged node-agent has crashed this host.
+KS_POST_RENDERER := ./kubescape/force-network-streaming.sh
+
 .PHONY: kubescape
-kubescape: 
+kubescape:
```

checkmake (0.3.2)

[warning] Target body for "kubescape" exceeds allowed length of 5 lines (9).

(maxbodylength)